Automattic/vip-go-mu-plugins
# Add library to implement cache variance

## Description
Allows developers to vary cached content for a single resource(page) by various methods

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
2. Add vary cache logic according to the example docs
3. Confirm you see different cached content in different contexts
